package.json

```diff
@@ -1,27 +1,27 @@
 {
   "dependencies": {
+    "axios": "^0.21.1",
+    "crypto-js": "^4.0.0",
     "url-join": "^4.0.1",
     "uuid": "^8.3.2",
     "validator": "^13.5.2",
-    "axios": "^0.21.1",
-    "crypto-js": "^4.0.0",
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "axios-mock-adapter": "^1.18.1",
     "body-parser": "^1.19.0",
     "cookie-parser": "^1.4.5",
     "express": "^4.17.1",
+    "fdk-client-javascript": "git+ssh://git@github.com/gofynd/fdk-client-javascript.git#0.1.8",
     "ioredis": "^4.24.2",
-    "jshint": "^2.12.0",
-    "ngrok": "^3.4.0",
-    "querystring": "^0.2.1",
-    "axios-mock-adapter": "^1.18.1",
     "jasmine": "^3.5.0",
     "jasmine-console-reporter": "^3.1.0",
+    "jshint": "^2.12.0",
+    "ngrok": "^3.4.0",
     "nyc": "^15.0.1",
+    "querystring": "^0.2.1",
```

Comment:
move it to normal dependancy if you are using package in actual code. In case of dependancies required for test cases and developer side changes should be here.